bus/bus.go

```diff
@@ -466,6 +470,26 @@ func (b *bus) contractsSpendingHandlerPOST(jc jape.Context) {
 	}
 }
 
+func (b *bus) hostsAllowlistHandlerGET(jc jape.Context) {
+	allowlist, err := b.hdb.HostAllowlist(jc.Request.Context())
+	if jc.Check("couldn't load allowlist", err) == nil {
+		jc.Encode(allowlist)
+	}
+}
+
+func (b *bus) hostsAllowlistHandlerPUT(jc jape.Context) {
+	ctx := jc.Request.Context()
+	var req api.UpdateAllowlistRequest
+	if jc.Decode(&req) == nil {
+		if jc.Check("couldn't add allowlist entries", b.hdb.AddHostAllowlistEntries(ctx, req.Add)) != nil {
+			return
+		}
+		if jc.Check("couldn't remove allowlist entries", b.hdb.RemoveHostAllowlistEntries(ctx, req.Remove)) != nil {
```

Comment:
Is there a reason you didn't combine them into a single method to match the endpoint?
The advantage would be that adding and removing could be a single atomic transaction. So when the endpoint returns an error, the user could rely on the allowlist not changing.